docs/avoid-sudo-linux.md

```diff
@@ -19,12 +19,29 @@ Make sure that it's effective by running `groups`. You might have to logout and
 
 ### 3. Make sure the uinput device file has the right permissions.
 
-Add a udev rule (in either `/etc/udev/rules.d` or `/lib/udev/rules.d`) with the following content:
+#### Create a new file name:
+`/etc/udev/rules.d/99-input.rules`
 
-```
+#### Insert the following in the code
+```bash
 KERNEL=="uinput", MODE="0660", GROUP="uinput", OPTIONS+="static_node=uinput"
 ```
 
+#### Machine reboot or run this to reload
+```bash
+sudo sh -c ‘udevadm control –reload; udevadm trigger -v –name-match uinput’
```

Comment:
Why is this `‘` being used instead of the regular <kbd>'</kbd> apostrophe? It's a an error if copy-pasted into a shell.